runtime.js

```diff
@@ -540,6 +540,13 @@ const _loadScript = async (file, {files = null, parentUrl = null} = {}) => {
     script = script.replace(r, function() {
       return arguments[1] + replacements[index++] + arguments[3];
     });
+    if (instanceId) {
+      script = script.replace("document.monetization", `window.document.monetization${instanceId}`);
+      script = script.replace("document.monetization.addEventListener", `window.document.monetization${instanceId}.addEventListener`);
+      script = `
+        window.document.monetization${instanceId} = window.document.createElement('div');
```

Comment:
Probably `new EventTarget()` is more logical than a DOM element.

Reply:
d383152